src/components/Login/style.scss

```diff
@@ -0,0 +1,176 @@
+.login-header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  height: 120px;
+  width: -webkit-fill-available;
+  left: 0px;
+  top: 0px;
+  box-shadow: 4px 4px 4px 4px rgba(0, 0, 0, 0.2);
+}
+
+.login-header {
```

Comment:
you don't need to write ".login-header" again, just write "&__text" inside ".login-header" block